tests/expressions_case/tests.py

```diff
@@ -21,6 +22,7 @@
     Image = None
 
 
+@ignore_warnings(category=RemovedInDjango20Warning)
```

Comment:
I believe I did this because a the [CaseTestModel](https://github.com/Brobin/django/blob/deprecate_csi_field/tests/expressions_case/models.py#L21) used in this method has a `CommaSeparatedIntegerField`. If its not needed I'll pull it out.